src/scripts/lint.ts

```diff
@@ -16,23 +14,17 @@ program
       'quality before committing or pushing changes.',
   )
   .option('-a, --all', 'lint all files in the repository')
```

Comment:
the flag is left here intentionally to not break jenkins CI.

Reply:
nit: let's change the description to something like "Does nothing, was left for a backward compatibility".

Reply:
I'll clean devops repo and remove it now